src/main.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+extern crate clap;
+
+use clap::{App, Arg, SubCommand};
+
+fn main() {
+    let matches = App::new("Mentat")
+                    .subcommand(SubCommand::with_name("serve")
+                                            .about("Starts a server")
+                                            .arg(Arg::with_name("debug")
+                                                .long("debug")
+                                                .help("Print debugging info"))
+                                            .arg(Arg::with_name("database")
+                                                 .short("d")
+                                                 .long("database")
+                                                 .value_name("FILE")
+                                                 .help("Path to the Mentat database to serve")
+                                                 .default_value("temp.db")
```

Comment:
An empty string here will behave as an in-memory database, which is probably a good default.